pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>local.pbaranowski</groupId>
+    <artifactId>zadanieJDK17</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <build>
```

Comment:
myślę że dobrym pomysłem byłoby tu jeszcze dorzucenie Lomboka

Reply:
Nie używałem, to nie dodawałem.